### PR TITLE
A proposal for FIP-0023 - Updating the BP Pay reserves to ensure payments for a couple of months

### PR DESCRIPTION
**Pull request by sukeshtedla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ FIPs describe proposed changes to the FIO Protocol.
 |[FIP-20](fip-0020.md)|FIO Co-op|Deferred|
 |[FIP-21](fip-0021.md)|FIO Staking|Deferred|
 |[FIP-22](fip-0022.md)|Retire FIO Tokens|Accepted|
+|[FIP-X](fip-0023.md)|Temporary adjustment of reserves to pay Block Producers|Draft|
```
Comment (lukestokes) [inline README.md:29]: Fantastic! Go ahead and update this to FIP 23.

File: fip-0023.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+fip: X
```
Comment (lukestokes) [inline fip-0023.md:2]: This is needed, so let's go ahead and give a FIP number. 23 is next in line so feel free to put that in here. Thanks!

File: fip-0023.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+This proposal is aimed at implementing a short-term solution to address the issue now and to secure some more time to work on a permanent solution.
+
+* For more detailed discusson on this topic, please visit: https://fioprotocol.atlassian.net/wiki/spaces/FC/pages/49184971/The+Future+of+Block+Producer+Reserves 
+
+Proposed new actions:
+Contract|Action|Description|
+|---|---|---|
+|fio.treasury|Updating BPMAXRESERVE|Need to update the BPMAXRESERVE value from 10000000000000000(10M Tokens) to 15000000000000000(15M Tokens) |
+
+# Motivation
```
Comment (lukestokes) [inline fip-0023.md:22]: Can you add some details here? Looks like it's currently just the template. Maybe show some info on what BP costs are and how the current fee-based approach without any subsidization from the foundation at this time isn't quite enough to keep BPs engaged and working on FIO.

**Review by lukestokes — CHANGES_REQUESTED** [review]

**Comment by lukestokes** [comment]
Also, could you add some thoughts on where the 5M tokens will come from? Should we take from the address giveaway bucket? Should the 38M tokens that are currently locked forever be burned and if so, how should we code that up?

**Comment by adsorptionenthalpy** [comment]
Wherever the tokens do end up coming from, their emittance should decline over time so the rewards last longer than a few months. Would you care to spec out any of these approaches as a part of this FIP? @sukeshtedla 
https://fioprotocol.atlassian.net/wiki/spaces/FC/pages/49184971/The+Future+of+Block+Producer+Reserves

**Comment by lukestokes** [comment]
> Wherever the tokens do end up coming from, their emittance should decline over time so the rewards last longer than a few months. Would you care to spec out any of these approaches as a part of this FIP? @sukeshtedla
> https://fioprotocol.atlassian.net/wiki/spaces/FC/pages/49184971/The+Future+of+Block+Producer+Reserves

I think the plan on this was just to create something we could deploy quickly as a three month window to then plan out something more advanced.

File: fip-0023.md (hunk 3 above)
Comment (sukeshtedla) [reply fip-0023.md:22]: I added the details now.

**Review by lukestokes — APPROVED** [review]

**Comment by lukestokes** [comment]
I'm going to approve this for now as a draft. I think we still need to add code to actually describe how the tokens will be burned to cover these increases.

**Pull request merged**